src/button.scss

```diff
@@ -220,7 +221,7 @@ a.#{$block} {
 
       // placements
       &::before {
-        margin-right: fd-space(2);
+        margin-right: $fd-button-icon-margin-right;
```

Comment:
Could you also apply these changes to RTL mode?

Reply:
sure

Reply:
fixed